src/components/MediaPlayer/MainControl/PlayerButtons/PlayerButtons.js

```diff
@@ -25,6 +25,8 @@ export class PlayerButtonsComponent {
   finishPlay() {
     this.isPlayNow = false;
     this.playButton.children[0].innerText = "play_arrow";
+    this.props.onPlayerChangeState(false);
+    this.props.onNextClick();
```

Comment:
I think it is better to loop the playlist, so the next song starts automatically. What do you think? @ihor4568